src/statsig/index.test.ts

```diff
@@ -0,0 +1,46 @@
+import { Statsig } from 'statsig-react-native'
+import { getExperimentParams } from 'src/statsig/index'
+import { StatsigExperiments } from 'src/statsig/types'
+import Logger from 'src/utils/Logger'
+
+jest.mock('statsig-react-native')
+jest.mock('src/utils/Logger')
+
+describe('Statsig helpers', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+  describe('getExperimentParams', () => {
+    it('returns default values if getting statsig experiment throws error', () => {
+      ;(Statsig.getExperiment as jest.Mock).mockImplementation(() => {
+        throw new Error('mock error')
+      })
+      const defaultValues = { param1: 'defaultValue1', param2: 'defaultValue2' }
+      const experimentName = StatsigExperiments.RECOVERY_PHRASE_IN_ONBOARDING // chosen arbitrarily
```

Comment:
nit: maybe mock constants/types and use a test specific experiment if its easy to do? We will likely remove experiment specific code in the future right?